libraries/testing/tester.cpp

```diff
@@ -1127,13 +1127,18 @@ namespace eosio { namespace testing {
       vector<digest_type> preactivations;
 
       std::function<void(const digest_type&)> add_digests =
-      [&pfm, &pfs, current_block_num, current_block_time, &preactivation_set, &preactivations, &add_digests]
+      [&ignored_features, &pfm, &pfs, current_block_num, current_block_time, &preactivation_set, &preactivations, &add_digests]
       ( const digest_type& feature_digest ) {
          const auto& pf = pfs.get_protocol_feature( feature_digest );
          FC_ASSERT( pf.builtin_feature, "called add_digests on a non-builtin protocol feature" );
          if( !pf.enabled || pf.earliest_allowed_activation_time > current_block_time
              || pfm.is_builtin_activated( *pf.builtin_feature, current_block_num ) ) return;
 
+         const auto feature_is_ignored = std::find(ignored_features.cbegin(), ignored_features.cend(), feature_digest);
+         if( feature_is_ignored != ignored_features.cend() ) {
+            return;
+         }
```

Comment:
This won't work correctly if you ignore a feature that has other features that depend on it.

Reply:
I'm trying a different approach in my latest commit.